### PR TITLE
[SDK-3291] Add interactive Laravel API QuickStart

### PR DESCRIPTION
**Pull request by frederikprijck**
image

### CODE REVIEW

File: articles/quickstart/backend/laravel/interactive.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+   path: app
+interactive: true
+files:
+  - files/auth
+  - files/env
+  - files/web
+
+---
+
+# Add authorization to a Laravel application.
```
Comment (stevehobbsdev) [inline articles/quickstart/backend/laravel/interactive.md]:
```suggestion
# Add authorization to a Laravel application
```

File: articles/quickstart/backend/laravel/interactive.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+This guide demonstrates how to integrate Auth0 with any new or existing Laravel API application using the [Auth0 Laravel SDK](https://github.com/auth0/laravel-auth0).
+
+If you haven't created an API in your Auth0 dashboard yet, you can use the interactive selector to create a new Auth0 API or select an existing API that represents the project you want to integrate with.
+
+Alternatively, you can read [our getting started guide](get-started/auth0-overview/set-up-apis) that helps you set up your first API through the Auth0 dashboard.
+
+Every API in Auth0 is configured using an API Identifier that your application code will use as the Audience to validate the access token.
```
Comment (stevehobbsdev) [inline articles/quickstart/backend/laravel/interactive.md]:
```suggestion
Every API in Auth0 is configured using an API identifier that your application code will use as the audience to validate the access token.
```

File: articles/quickstart/backend/laravel/interactive.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+
+<%= include('../../../_includes/_api_auth_intro') %>
+
+## Define permissions
+<%= include('../_includes/_api_scopes_access_resources') %>
+
+## Create a Laravel Application
```
Comment (stevehobbsdev) [inline articles/quickstart/backend/laravel/interactive.md]:
```suggestion
## Create a Laravel application
```

File: articles/quickstart/backend/laravel/interactive.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+
+::: note
+If you already have a Laravel 9 application prepared, you can skip this step.
+:::
+
+Let's begin by setting up a new Laravel application. Let's open a shell and run the following command — replacing `DIRECTORY_NAME` with a directory name of preference to create and install Laravel within. The directory cannot already exist.
```
Comment (stevehobbsdev) [inline articles/quickstart/backend/laravel/interactive.md]:
```suggestion
Let's begin by setting up a new Laravel application. Open a shell and run the following command — replacing `DIRECTORY_NAME` with a directory name of preference to create and install Laravel within. The directory cannot already exist.
```

File: articles/quickstart/backend/laravel/interactive.md [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+  "url": "http://localhost:3010/api/private",
+  "headers": [
+    { "name": "Authorization", "value": "Bearer YOUR_ACCESS_TOKEN" }
+  ]
+}
+```
+
+### Obtain an access token
+
+If you are calling the API from a Single-Page Application or a Mobile/Native application, after the authorization flow is completed, you will get an access token. How you get the token and how you make the call to the API will be dependent on the type of application you are developing and the framework you are using. For more information refer to the relevant application Quickstarts which contain detailed instructions:
```
Comment (stevehobbsdev) [inline articles/quickstart/backend/laravel/interactive.md]:
```suggestion
If you are calling the API from a single-page application or a mobile/native application, after the authorization flow is completed, you will get an access token. How you get the token and how you make the call to the API will be dependent on the type of application you are developing and the framework you are using. For more information refer to the relevant application quickstarts which contain detailed instructions:
```

File: articles/quickstart/backend/laravel/interactive.md [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+
+* [Single-Page Applications](/quickstart/spa)
```
Comment (stevehobbsdev) [inline articles/quickstart/backend/laravel/interactive.md]:
```suggestion
* [Single-page applications](/quickstart/spa)
```

File: articles/quickstart/backend/laravel/interactive.md [hunk 7]
```diff
@@ -0,0 +1,176 @@
...
+* [Mobile / Native Application](/quickstart/native)
```
Comment (stevehobbsdev) [inline articles/quickstart/backend/laravel/interactive.md]:
```suggestion
* [Mobile / native application](/quickstart/native)
```

File: articles/quickstart/backend/laravel/interactive.md [hunk 8]
```diff
@@ -0,0 +1,176 @@
...
+        "name": "audience",
+        "value": "YOUR_API_IDENTIFIER"
+      }
+    ]
+  }
+}
+```
+
+:::note
+Auth0 customers are billed based on the number of Machine to Machine access tokens issued by Auth0. Once your application gets an access token it should keep using it until it expires, to minimize the number of tokens requested.
```
Comment (stevehobbsdev) [inline articles/quickstart/backend/laravel/interactive.md]:
```suggestion
Auth0 customers are billed based on the number of machine-to-machine access tokens issued by Auth0. Once your application gets an access token it should keep using it until it expires, to minimize the number of tokens requested.
```

**Review by stevehobbsdev — CHANGES_REQUESTED** [review]
👍🏻

**Review by stevehobbsdev — DISMISSED** [review]

**Review by evansims — DISMISSED** [review]

**Review by avanscoy — APPROVED** [review]

**Pull request merged**